Formula/subversion.rb

```diff
@@ -126,6 +116,9 @@ def install
       --enable-optimize
       --with-zlib=/usr
       --with-sqlite=#{Formula["sqlite"].opt_prefix}
+      --with-apr=#{Formula["apr"].opt_prefix}
+      --with-apr-util=#{Formula["apr-util"].opt_prefix}
+      --with-apxs=no
```

Comment:
Mmm. Don't quote me on this because I'm working from memory, but I believe `apxs` would use the system's `apxs` which links/references to the system's Apr, and you'd have overlapping Apr linkage.